packages/grid/src/GridRenderer.ts

```diff
@@ -511,30 +594,39 @@ class GridRenderer {
           ? visibleColumnXs.get(floatingLeftColumnCount + 1)
           : -10,
       minY:
-        floatingTopColumnCount > 0 &&
-        visibleRowYs.has(floatingTopColumnCount + 1)
-          ? visibleRowYs.get(floatingTopColumnCount + 1)
+        floatingTopRowCount > 0 && visibleRowYs.has(floatingTopRowCount + 1)
+          ? visibleRowYs.get(floatingTopRowCount + 1)
```

Comment:
Because those ranges may not actually be visible, they may be off screen

Reply:
Errrrr wait. I'm already checking for `.has` so the getOrThrow will never throw anyway - I think it's because if it's undefined then the addition operation in the below would give an error, so I wasn't getting an error here. I'll add it just to be consistent